src/js/angular/rest/rdf4j.repositories.rest.service.js

```diff
@@ -134,7 +134,7 @@ function RDF4JRepositoriesRestService($http, $repositories, $translate) {
         const payloadString = properties.join('&');
         return $http({
             method: 'POST',
-            url: `/repositories/${repositoryId}`,
+            url: `repositories/${repositoryId}`,
```

Comment:
Can you please try using the `REPOSITORIES_ENDPOINT` constant here as well.

Reply:
Done. Thanks.